src/utils/config.js

```diff
@@ -11,3 +11,6 @@ export const cartoHousingActionsTable = "public.emergency_housing_actions";
 export const cartoCountiesTable = "public.us_county_boundaries";
 export const cartoStatesTable = "public.states_and_provinces_global";
 export const cartoNationsTable = "public.countries";
+
+export const bingApiKey =
+  "Al2-1GXrd8GzwtSAID3J3LJn-flLLgNWzNtsT5nnSKW8dA2ClgaXXXMQR6WfE6wE";
```

Comment:
@jimboweb perhaps we can keep this as a github secret key instead of making it public?  Here is a link on how to do that. https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository

Reply:
As far as I know, there's no way to truly hide it as long as it's a frontend app. We can store it on the server and retrieve it somehow, but people will still be able to look at the network traffic and see what it is. We could obfuscate it better.

The only way to truly hide it is to do the requests from the backend. As far as I know.

Reply:
This person has a solution that might work: 

https://blog.mauriciorobayo.com/how-to-hide-your-front-end-api-keys-ck6jqvll900y3dfs11u3x96tn

Reply:
@jimboweb there probably is a way you can limit what URLs can access the API using this key in your Bing developer account settings. This is typically what is done to limit API keys that are exposed publicly.